code-challenges.test.js

```diff
@@ -166,11 +162,11 @@ describe("noDupes", () => {
 // // b) Create the function that makes the test pass.
 
 // //**PSEUDOCODE****************
-// //create function - arr3
+// //create function called noDupes
 // //parameters will be array1/array2
-// //thinking to join arrays together
-// //use a method  to remove similar values
-// //data type will be new array of non duplicate values
+// //thinking to pass arrays together in the function
+// //use a method .set to remove similar values
+// //data type will be new array of non duplicate values using spread operator
```

Comment:
Great!

Reply:
sorry i thought i updated it....

//create a function called miniMax
//takes in an array
//need a way to get the minimum and maximum values in each array
//at the same time not returning the remaining values
//found that i can use math.min - math.max method to get those values
//with a bit a of help found out that the spread operator will include all the elements 
//to determine the min/max elements(i believe)
//return an array with new numbers using min/max
// // it worked!